src/main.js

```diff
@@ -0,0 +1,42 @@
+function strComp (str1, str2) {
+  if (str1.localeCompare(str2) === -1 || str1.localeCompare(str2) === 0){
+    return true;
+  }
+}
+
+function checkAndPush(inputElement, liArray) {
+  const tempLiItem = document.createElement("li");
+  tempLiItem.textContent = inputElement.name;
+  for (j=0; j<liArray.length; j++){
+    if (strComp(inputElement.name.toUpperCase(), liArray[j].textContent.toUpperCase()))
+      break;
+  }
+  return j;
+}
+
+function renderInput(input, container){
+  const dirItem = [];
+  const fileItem = [];
+
+  const dirList = document.createElement("ul");
```

Comment:
We should use one list here. We want both directories and files in the same list (as described in #1).